app/src/main/java/com/gholem/moneylab/features/add/navigation/AddNavigation.kt

```diff
@@ -0,0 +1,16 @@
+package com.gholem.moneylab.features.add.navigation
+
+import com.gholem.moneylab.arch.nav.NavControllerWrapper
+import com.gholem.moneylab.arch.nav.NavigationController
+import javax.inject.Inject
+
+class AddNavigation @Inject constructor(
```

Comment:
I suggest to change the name to `AddTransactionNavigation` and also the `AddFragment `to `AddTransactionFragment `(all corresponding classes also ie. `ViewModel`)